Missionframework/functions/fn_createCrew.sqf

```diff
@@ -0,0 +1,86 @@
+/*
+	File: fn_createCrew.sqf
+	Author: PiG13BR - https://github.com/PiG13BR/
+	Date: 2024-26-08
+	Last Update: 2024-27-08
+	License: MIT License - http://www.opensource.org/licenses/MIT
+
+	Description:
+		Creates vehicle crew based on preset configuration only
+	
+	Parameter(s):
+		_vehicle - Vehicle to add crew
+		_side - crew side: KPLIB_side_enemy or KPLIB_side_player (Default: KPLIB_side_enemy)
+
+	Returns:
+		Array - [Crew created, group]
+*/
+
+params[
+	["_vehicle", objNull, [objNull]],
+	["_side", KPLIB_side_enemy, [sideUnknown]]
+
+];
+
+if (isNull _vehicle) exitWith {["No vehicle provided"] call BIS_fnc_error};
+
+private _typeCrew = "";
+
+// Placeholder types
+switch (_side) do {
+	case KPLIB_side_enemy : {
+		 _typeCrew = KPLIB_o_rifleman;
+	};
+    case KPLIB_side_player : {
+    	_typeCrew = KPLIB_b_crewUnit;
+    }
+};
+
+/*
+Save it for later changes
+switch (_side) do {
+	case KPLIB_side_enemy : {
+		if (_vehicle isKindOf "Air") then {
+			_typeCrew = KPLIB_o_pilot;
+		} else {
+			if ((_vehicle isKindOf "Tank") || {_vehicle isKindOf "Wheeled_APC_F"}) then {
+				_typeCrew = KPLIB_o_crewman;
+			} else {
+				_typeCrew = KPLIB_o_rifleman;
+			}
+		}
+	};
+	case KPLIB_side_player : {
+		if (_vehicle isKindOf "Air") then {
+			_typeCrew = KPLIB_b_heliPilotUnit;
+		} else {
+			if ((_vehicle isKindOf "Tank") || {_vehicle isKindOf "Wheeled_APC_F"}) then {
+				_typeCrew = KPLIB_b_crewUnit;
+			} else {
+				_typeCrew = KPLIB_b_crewStatic;
+			}
+		}
+	}
+};
+*/
```

Comment:
Is this supposed to be commented out?

Reply:
> Is this supposed to be commented out?

If you want to add those new variables in presets, then yes. The original version uses them. I tested a lot and works just fine.